src/sage/modular/multiple_zeta.py

```diff
@@ -248,7 +250,7 @@ def coproduct_iterator(paire):
                                            tail[step:]))
 
 
-def composition_to_iterated(w, reverse=False):
+def composition_to_iterated(w, reverse=False) -> tuple[int, ...]:
```

Comment:
This annotation requires Python >= 3.9, but we still support 3.8;
see https://docs.python.org/3/library/typing.html#typing.Tuple

Reply:
even with the added line `from __future__  import annotations` ?

Reply:
We already have the exact same annotation in `src/sage/combinat/interval_posets.py` and other places.

Reply:
OK, thanks.